packages/wordcount/package.json

```diff
@@ -25,5 +25,11 @@
 	},
 	"publishConfig": {
 		"access": "public"
+	},
+	"devDependencies": {
+		"@wordpress/docgen": "file:../docgen"
+	},
+	"scripts": {
```

Comment:
This part is not clear to me. Ideally, it's handled the same way for all packages in the root of the repository to avoid all hassle. At the moment, we would have to repeat the same steps for over 50 packages. It's easy to identify the starting point as it is always defined in `main` entry in the `package.json` file or defaults to `index.js`.

Reply:
What if we detect if `README` file contains `<!-- START TOKEN(Autogenerated API docs) -->` and then start the process for each of the existing packages? This way we could introduce changes gradually from one place.

Reply:
I'm still unclear on our lerna setup! This is what I know: `lerna run x` executes the `x` command in every package that has it, but the packages still have to declare that script. From what you mention, should I understand that we prefer to bypass this lerna mechanism and develop a `bin` script that is only runnable from the Gutenberg repo?

My reasoning to add `devDependencies` and the `script` to each package was that it'd play nicely with lerna and, at the same time, it'll make the command available if you are only using a single package.

Reply:
ok, prepared https://github.com/WordPress/gutenberg/pull/14216 as a starting point and will refactor this accordingly.

I'm still preparing some packages: some may need different configurations (`ignore` option, using `--to-section` instead of `--to-token`, etc). I'd rather start really simple and then refactor as we grow.